File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/ProcessingJobCreateAdjustProcessingJobRequestItemPackagesItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class ProcessingJobCreateAdjustProcessingJobRequestItemPackagesItem {
    @JsonProperty("Label")
    public String label;
    @JsonProperty("Quantity")
    public Double quantity;
    @JsonProperty("UnitOfMeasure")
    public String unitOfMeasure;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/ProcessingJobCreateAdjustProcessingJobRequestItemPackagesItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class ProcessingJobCreateAdjustProcessingJobRequestItemPackagesItem(
    @JsonProperty("Label")
    val label: String? = null,
    @JsonProperty("Quantity")
    val quantity: Double? = null,
    @JsonProperty("UnitOfMeasure")
    val unitOfMeasure: String? = null
)
